alerts/feedback_events.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2015 Mozilla Corporation
+#
+# Contributors:
+# Brandon Myers <bmyers@mozilla.com>
+
+from lib.alerttask import AlertTask
+from query_models import SearchQuery, TermMatch
+
+
+class AlertFeedbackEvents(AlertTask):
+    def main(self):
+        self._config = self.parse_json_alert_config('feedback_events.json')
+        search_query = SearchQuery(minutes=30)
+
+        search_query.add_must([
+            TermMatch('_type', 'event'),
```

Comment:
I think we should remove the match on _type since we are moving away from using this as a term we match on to align with the migration.

Reply:
Good point, fixed in https://github.com/mozilla/MozDef/pull/575/commits/167a3d6374b779cb9d33a60f1dc3c6f80faa61e6